client/src/components/AskQuestion.jsx

```diff
@@ -14,17 +13,69 @@ const TextField = styled.textarea `
 TextField.displayName = 'TextField';
 
 const SubmitQButton = styled.button`
+  background-color: rgb(0, 0, 0);
+  border-color: rgb(0, 0, 0);
+  border-radius: 3px;
+  border-width: 1px;
+  display: block;
+  color: rgb(255, 255, 255);
+  // Will need to fix float
+  padding-bottom: 8px;
```

Comment:
just heads up you can do all of padding for all 4 sides in just one css property! this works for margin and border (and i think a few other things too) https://developer.mozilla.org/en-US/docs/Web/CSS/padding

Reply:
Yeah lol I know I've been frantically doing css I should clean it up next week